tests/NATS.Client.Core.Tests/NatsHeaderTest.cs

```diff
@@ -177,4 +177,21 @@ public void ParserMultiSpanTests()
         Assert.Equal(123, headers.Code);
         Assert.Equal("Test Message", headers.MessageText);
     }
+
+    [Theory]
+    [InlineData("NATS/1.0 123 Test Message\r\nk2: v2-0\r\nk2: v2-1\r\n\r\n", "v2-1")]
+    [InlineData("NATS/1.0 123 Test Message\r\nk2: v2-0\r\n\r\n", "v2-0")]
+    public void GetLastValueTests(string text, string expectedLastValue)
+    {
+        var parser = new NatsHeaderParser(Encoding.UTF8);
+        var input = new SequenceReader<byte>(new ReadOnlySequence<byte>(Encoding.UTF8.GetBytes(text)));
+        var headers = new NatsHeaders();
+        parser.ParseHeaders(input, headers);
+
+        Assert.True(headers.ContainsKey("k2"));
+
+        var result = headers.TryGetLastValue("k2", out var lastValue);
+        Assert.True(result);
+        Assert.Equal(lastValue, expectedLastValue);
+    }
```

Comment:
could you also add a test where key doesn't exist?

Reply:
Yes, added